common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -52,7 +52,7 @@ class SequenceFields(object):
             "Note, you must enable Entrance Exams for this course setting to take effect."
         ),
         default=False,
-        scope=Scope.content,
+        scope=Scope.settings,
```

Comment:
Just one comment, AFAIK, this is a backwards incompatible change. It won't affect edx.org website, as I don't think we are using it, but we may want to check with out customer(s) and see if it is in use.